src/pybind/mgr/dashboard/controllers/rbd.py

```diff
@@ -76,6 +76,7 @@ def _rbd_list(self, pool_name):
             features = i.features()
             stat['features'] = features
             stat['features_name'] = self._format_bitmask(features)
+            stat['parent_name'] = stat['parent_name'].decode('utf8')
```

Comment:
Interesting. this seems like a problem of the `rbd` python binding to me.

Reply:
good point! will check rbd python binding implementation

Reply:
I checked the rbd python binding implementation and the `parent_name` and `parent_pool` are already deprecated, so I changed the code to remove those two keys from the dictionary.